src/utils/output.js

```diff
@@ -15,8 +15,10 @@ const { getLinks, getAssertionResults } = require('./lhci-helpers')
 exports.setOutput = async function setOutput(resultsPath) {
   const links = await getLinks(resultsPath)
   const assertionResults = await getAssertionResults(resultsPath)
+  const manifestResults = await getManifest(resultsPath)
 
   core.setOutput('resultsPath', resultsPath)
   core.setOutput('links', links ? JSON.stringify(links) : '')
   core.setOutput('assertionResults', assertionResults ? JSON.stringify(assertionResults) : '')
+  core.setOutput('manifest', manifestResults ? JSON.stringify(manifestResults) : '')
```

Comment:
or do we prefer `reports` instead?

Reply:
I think, it's better to match LHCI namings.